stable_baselines/common/vec_env/base_vec_env.py

```diff
@@ -127,6 +127,17 @@ def env_method(self, method_name, *method_args, indices=None, **method_kwargs):
         """
         pass
 
+    @abstractmethod
+    def seed(self, seed=None):
```

Comment:
We started typing everything recently so please use type hints ;)
```suggestion
    def seed(self, seed: Optional[int] = None) -> List[int]:
```
Note: you will need to import `Optional` and `List` from the typing module and update the docstring too

EDIT: are you sure the return type is `List[int]`? I thought it would be `None`

Reply:
Is it sufficient to use type hints only in the base class?